documentation/library_guide/extension_api.rst

```diff
@@ -313,3 +313,57 @@ Example of Range-based API usage
 
         copy(oneapi::dpl::execution::dpcpp_default, view, range_res);
     }
+
+Async API
+--------------------------
+
+Functions as defined in the STL algorithm or numeric headers are traditionally blocking. oneDPL extends the functionality of C++17 parallel algorithm by asynchronous algorithm with non-blocking behavior. This experimental feature enables expressing concurrent control flow by building dependency chains and interleaving algorithm calls, as well as interoperability with SYCL kernels. The following subset of C++17 STL algorithms are currently supported as async variants:
+
+* ``reduce``
```

Comment:
I'd suggest to have alphabetical order for algorithms to seek an algorithm faster by user

Reply:
Yes makes sense. done.